Refactor Seq2SeqModelCaffe2EnsembleDecoder

pritamdamania87 opened the pull request:
Summary: This is an initial diff which refactors some of the components in the Seq2SeqModelCaffe2EnsembleDecoder class.

Differential Revision: D9026372

jmp84 commented:
@pritamdamania87, looks good, but you probably want to rebase and reexport PR (or alternatively write a comment like "@pytorchbot retest this please") since CI is failing.

jmp84 commented:
Oh, it's not failing, we just need to wait more:)

pritamdamania committed:
Refactor Seq2SeqModelCaffe2EnsembleDecoder (#10035)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/10035

This is an initial diff which refactors some of the components in the Seq2SeqModelCaffe2EnsembleDecoder class.

Differential Revision: D9026372

fbshipit-source-id: 8be2fc701230578dafb5de0e57f468605fdcfb17
